services/data-collection/autocapture.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: Autocapture
+description: Automatically capture events from your website.
+icon: wand-sparkles
+---
+
+import EdgeeSdk from '/snippets/edgee-sdk.mdx';
+
+<EdgeeSdk />
+
+Autocapture allows you to automatically collect various user interactions and events from your website without writing any custom code. 
+By enabling different autocapture settings, you can gather valuable analytics data with minimal setup.
+
+## Configuration
+
+You can configure autocapture settings in your project settings. Each feature can be enabled or disabled independently to match your specific needs.
+
+<Frame caption="Autocapture settings in your project dashboard">
+  <img src="/images/autocapture.png" alt="Autocapture Settings" />
+</Frame>
+
+### Available Features
+
+#### Pageview Tracking
+Automatically captures when users visit your pages. This feature is enabled by default and provides essential analytics about:
+- Page visits
+- Sessions
+- User navigation patterns
+- Entry and exit pages
+
+#### SPA Pageview Tracking
+Specifically designed for Single Page Applications (SPAs), this feature automatically captures page views by intercepting:
+- `history.pushState()` calls
+- `history.replaceState()` calls
+- Browser back/forward navigation (`popstate` events)
+
+<Warning>
+You don't need to enable this feature if you're manually integrating the [Edgee React SDK](/getting-started/react-sdk).
+</Warning>
+
+#### Engagement Tracking
+Measures how users actively engage with your pages by tracking:
+- Tab visibility changes (when users switch between tabs)
+- Window minimize/maximize events
+- Page unload events
+- Time spent actively engaging with your content
+
+The engagement time is calculated from the moment the page loads until the user leaves or switches tabs.
+
+#### Click Tracking
+Automatically captures interactions with clickable elements including:
+- Buttons (`<button>`)
+- Links (`<a>`)
+- Form elements (`<form>`)
+- Input fields (`<input>`)
+- Select dropdowns (`<select>`)
+- Labels (`<label>`)
+
+For each click event, it collects:
+- Element tag name (`target-tag`)
+- Element ID (`target-id`)
+- Element classes (`target-class`)
+- Text content (`target-text`)
+- href attributes for links (`target-href`)
+- Input values (`target-value`)
+
+<Note>
+Click tracking respects the `data-edgee-no-autocapture` attribute. Any element with this attribute or any of its parents will be excluded from click tracking.
```

Comment:
the SDK is doing this:

```js
parent.classList.contains('edgee-no-autocapture'))
```

So it should be a class named `edgee-no-autocapture` (instead of an attribute named `data-edgee-no-autocapture`), right?

Reply:
Also, this applies to form submission tracking as well.

Reply:
I just changed it. It could be both, classname and attribute, on the clicked element, or on his parent